pkg/skaffold/deploy/labels.go

```diff
@@ -194,3 +192,11 @@ func groupVersionResource(disco discovery.DiscoveryInterface, gvk schema.GroupVe
 
 	return schema.GroupVersionResource{}, fmt.Errorf("Could not find resource for %s", gvk.String())
 }
+
+func copyMap(dest, from map[string]string) {
+	if from != nil {
```

Comment:
you can remove this check

ranging over a nil map should be fine, it just will iterate 0 times

https://groups.google.com/forum/#!topic/golang-nuts/2TVM3Rg8w_w